aiohttp/web_main.py

```diff
@@ -0,0 +1,43 @@
+from .web import run_app
```

Comment:
Okay...I wasn't sure about the layout.
I was trying to emulate some resemblance to a package level `__main__.py`.
Speaking of which could `web` be turned into a proper sub-package, or would that break backwards compatibility?